Fix a few typos (#205)
ACCESS-Hive/access-hive.github.io: docs/model_evaluation/data_processing.md

[**Documentation**][xskillscore-doc] |
[**Sources**][xskillscore-source]

> xskillscore is a Python library for computing a wide variety of skill metrics. Its typical application is in the verification of deterministic and probabilistic forecasts relative to observations.

Reviewer: ```suggestion
`xskillscore` is a Python library for computing a wide variety of skill metrics. Its typical application is to verify deterministic and probabilistic forecasts relative to observations.
```

Author: Thanks @navidcy! I'm okay with the idea to use backticks for Python packages, but I think I'll leave this for another PR as there are a few other places this would need to be changed also.
